openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -512,129 +522,3 @@ def _parse_analytics_param_for_course_id(request):
                     analytics=analytics
                 )
             )
-
-
-def shim_student_view(view_func, check_logged_in=False):
-    """Create a "shim" view for a view function from the student Django app.
-
-    UPDATE: This shim is only used to wrap `login_user`, which now lives in
-    the user_authn Django app (not the student app).
-
-    Specifically, we need to:
-    * Strip out enrollment params, since the client for the new registration/login
-        page will communicate with the enrollment API to update enrollments.
-
-    * Return responses with HTTP status codes indicating success/failure
-        (instead of always using status 200, but setting "success" to False in
-        the JSON-serialized content of the response)
-
-    * Use status code 403 to indicate a login failure.
-
-    The shim will preserve any cookies set by the view.
-
-    Arguments:
-        view_func (function): The view function from the student Django app.
-
-    Keyword Args:
-        check_logged_in (boolean): If true, check whether the user successfully
-            authenticated and if not set the status to 403. This argument is
-            used to test the shim by skipping this check.
-
-    Returns:
-        function
-
-    """
-    @wraps(view_func)
-    def _inner(request):  # pylint: disable=missing-docstring
-        # Call the original view to generate a response.
-        # We can safely modify the status code or content
-        # of the response, but to be safe we won't mess
-        # with the headers.
-        response = view_func(request)
-
-        # Most responses from this view are JSON-encoded
-        # dictionaries with keys "success", "value", and
-        # (sometimes) "redirect_url".
-        #
-        # We want to communicate some of this information
-        # using HTTP status codes instead.
-        #
-        # We ignore the "redirect_url" parameter, because we don't need it:
```

Comment:
**Reviewer Note:** It is unclear if any of these thoughts/comments should be moved somewhere, and what is still relevant.